tokio/src/sync/broadcast.rs

```diff
@@ -1369,3 +1386,45 @@ impl<'a, T> Drop for RecvGuard<'a, T> {
 }
 
 fn is_unpin<T: Unpin>() {}
+
+#[cfg(not(loom))]
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn receiver_count_on_sender_constructor() {
+        let count_of = |sender: &Sender<i32>| sender.shared.tail.lock().rx_cnt;
```

Comment:
I would move this to a real method since you're defining it twice.

Reply:
Oops, I missed that `Sender::receiver_count` is 100% equivalent to what I was doing.

Updated.